CFP.md

```diff
@@ -0,0 +1,37 @@
+# Call for Participation: Inclusive Web Devroom
+
+<b>We invite you to submit a talk about anything that relates to making the Web a more inclusive space!</b>
+
+Websites and the extended web technologies such as browsers, extensions, etc. are regularly used by everyone, but sometimes people can get left behind if a technology hasn't been designed in a way that is inclusive for _everyone_ and not just a specific group of people. We firmly believe in making the Web a more inclusive space and want to encourage others to do so as well. 
+
+## Possible topics
+
+Some possible topics to inspire your talk: 
+
+- Web accessibility
+- Privacy and Security 
+- Performance
+- Localization and Internationalization 
+- Sustainability
+- ... and more! Any other form of work being done to make a the Web more inclusive is welcome! 
+
+
+Anyone who is working on any open-source assistive technologies, or building tools to enhance privacy or improve localization, or any other open-source public interest technology that makes the web more inclusive and helps focus on digital rights are welcome to submit a talk for this Devroom.
+
+_Not sure if your talk is a good fit?_ We encourage you to submit it anyway! 
+
+
+## Examples of previous talks at FOSDEM that would fit this Devroom
```

Comment:
These examples I got through a quick search from fosdem archives -- we can remove any that aren't relevant or add others

Reply:
They LGTM, if I had time I would trawl through and find more examples but maybe @SaptakS knows some offhand.

Reply:
I think this is fine. I would even say we could put examples of talk outside of FOSDEM if they match the criteria, but I think this is fine for now.